README.Rmd

```diff
@@ -128,7 +128,7 @@ To create a forest the minimum data needed is an n by d input matrix (X) and an
 ```{r create-a-forest}
 X <- as.matrix(iris[,1:4])
 Y <- iris[[5L]]
-forest <- RerF(X, Y, seed = 1L, num.cores = 1L)
+forest <- RerF(X, Y, seed = 1L, num.cores = 1L, max.depth = ceiling(log2(nrow(X))))
```

Comment:
Do we care that outputs have changed in the readme.md (because defaults changed?) 

If readme outputs change, and assuming it doesn't lengthen run times at all, I am now thinking it would be better to undo the changes I made in this file setting the max.depth.  I don't think we have a desire to encourage people to modify these defaults to these (old) default values.

Reply:
Agreed, I've removed the `max.depth` arguments